### DISCUSSION TITLE
Feature to create issues in the project page

### DISCUSSION THREAD

**Discussion by markustoivonen** [community/community#10069] [Projects and Issues] [answered]
Currently it is not possible to create issues on the project page (at least not that I've found). Would be nice to have a button that opens up **modal, not a new tab** for issue creation.

**Comment by jaredsb** [comment, marked as answer]
Hi @markustoivonen you can create an Issue from a project by first [making a draft Issue and then converting it to an Issue](https://github.blog/changelog/2021-08-17-the-new-github-issues-08-17-update/).  

Also, you can [open a draft Issue in a modal view](https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/) now, and you can expect more support for Issues in the project coming soon!

**Reply by ghost** [reply]
Thanks
